server/src/template-linter.ts

```diff
@@ -0,0 +1,89 @@
+import { Diagnostic, DiagnosticSeverity, Files, TextDocument } from 'vscode-languageserver';
+import { uriToFilePath } from 'vscode-languageserver/lib/files';
+
+import * as path from 'path';
+import * as fs from 'fs';
+
+import Server from './server';
+
+export default class TemplateLinter {
+
+  private _linter = {};
```

Comment:
I changed it to use a Map. Keys in WeakMaps have to be objects but we have to use a string here.